sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/functions.scala

```diff
@@ -546,6 +546,9 @@ case class HyperLogLogPlusPlus(child: Expression, relativeSD: Double = 0.05)
     AttributeReference(s"MS[$i]", LongType)()
   }
 
+  override val inputAggBufferAttributes: Seq[AttributeReference] =
```

Comment:
So, if anyone tries to move it back to `ImperativeAggregate`, he/she will see an test failure? Should we add a comment to `ImperativeAggregate` to explain why a leaf class needs to implement this?